lib/vmdb/console_methods.rb

```diff
@@ -41,5 +41,12 @@ def simulate_queue_worker(break_on_complete: false, quiet_polling: true)
         break if break_on_complete.kind_of?(Integer) && (break_on_complete -= 1) <= 0
       end
     end
+
+    # Helper method to set up a $evm variable for debugging purposes in Rails console
+    def initialize_automate_evm(user: nil)
```

Comment:
I saw @lfu suggestion for `automate_handle`.  I would vote for prefixing the name with `automate` because tab-completion in Rails console helps with the rest.

For example, `automate_initialize_evm` just requires me to type `autom<tab>`.  (I do the same with `simulate_queue_worker`.  It's just `simu<tab>`)

Reply:
I love the autocomplete idea.  Not sure about automate_handle specifically, but I'll change it to start with `automate_`